sleap/gui/dialogs/merge.py

```diff
@@ -40,9 +40,28 @@ def __init__(self, base_labels: Labels, new_labels: Labels, *args, **kwargs):
 
         super(MergeDialog, self).__init__(*args, **kwargs)
 
+        layout = QtWidgets.QVBoxLayout()
+
         self.base_labels = base_labels
         self.new_labels = new_labels
 
+        if self.base_labels.skeleton.node_names != self.new_labels.skeleton.node_names:
```

Comment:
node_names returns a list, so this comparison won't work unless they're ordered right?